### DISCUSSION TITLE
Where in the docs could a manual about YT videos fit?

### DISCUSSION THREAD

**Discussion by c1972** [thorsten/phpMyFAQ#3023] [Q&A] [answered]
Displaying a YouTube video in an FAQ post is pretty easy...

It gets a bit more complicated if you want to **automatically display the most recent video** from a YT channel.

I have written a manual for this ( See: https://forum.phpmyfaq.de/viewtopic.php?f=2&t=43446 ) and would like to add it to the docs.

My question: **Where in the docs would this manual fit?**

Thanks + regards Christian

**Comment by thorsten** [comment, marked as answer]
I'll add a section for e.g. "Handling content", what do you think?

**Comment by c1972** [comment]
😊 Thanks for asking! Sounds like a good solution to me.

Videos are often used and auto the most recent video... I think something like that is used from time to time when people see it there.
